addons/calendar/migrations/8.0.1.0/post-migration.py

```diff
@@ -92,11 +98,12 @@ def import_crm_meeting(cr):
             openupgrade.get_legacy_name('crm_meeting_id'),
         )
     )
-    # restore the disabled unique constraint
-    cr.execute(
-        '''alter table meeting_category_rel
-        add constraint meeting_category_rel_event_id_type_id_key
-        UNIQUE(event_id, type_id)''')
+    if constraint_enabled:
```

Comment:
It wasn't here, and now that I've checked it, not with
``` 
with cr.rollback():
        cr.execute(...)
```
but [here](https://github.com/OCA/l10n-spain/blob/deb2fb19364e584cc9162449c457f88f46d4d498/l10n_es_partner/migrations/8.0.1.3/pre-migration.py#L43), which is slightly different, but relies too in rollbacks.
The delete query in that line caused the internal error, while executing it manually before the rollback was fine.